README.rst

```diff
@@ -17,12 +17,39 @@ System Requirements
 
 Working on this repository requires that both Docker and Gradle are installed.
 
-install gradle
+Install gradle
 https://gradle.org/install/
 
-install docker
+Install docker
 https://docs.docker.com/install/
 
+Install firefox-geckodriver (if on Ubuntu)
+.. code:: bash
+
+    sudo apt install firefox-geckodriver
```

Comment:
^ good catch. This is a blindspot on my end. Everyone who had run these in the past was a member of DX and had this tooling set up.